x-pack/plugins/cloud/public/plugin.ts

```diff
@@ -198,7 +204,22 @@ export class CloudPlugin implements Plugin<CloudSetup> {
       if (userId) {
         // Do the hashing here to keep it at clear as possible in our source code that we do not send literal user IDs
         const hashedId = sha256(userId.toString());
-        fullStory.identify(hashedId);
+        const kibanaVer = this.initializerContext.env.packageInfo.version.split('.');
+        application?.then(async () => {
+          const appStart = await application;
```

Comment:
I know the promise is unlikely to reject, but it is still an unhandled promise rejection